cpp/src/plasma/malloc.cc

```diff
@@ -40,7 +42,7 @@ int fake_munmap(void*, int64_t);
 #define USE_DL_PREFIX
 #define HAVE_MORECORE 0
 #define DEFAULT_MMAP_THRESHOLD MAX_SIZE_T
-#define DEFAULT_GRANULARITY ((size_t)128U * 1024U)
+#define DEFAULT_GRANULARITY ((size_t) 1024U * 1024U * 1024U) // 1GB
```

Comment:
my plan is to change it back to the default if hugepages are not active (unfortunately this will be done only during compile time, since it is a macro and we can't set it at runtime)